sample/Directory.Packages.props

```diff
@@ -5,7 +5,7 @@
     <PackageVersion Include="Ardalis.ListStartupServices" Version="1.1.4" />
     <PackageVersion Include="Ardalis.Result" Version="10.0.0" />
     <PackageVersion Include="Ardalis.Result.AspNetCore" Version="10.0.0" />
-    <PackageVersion Include="Ardalis.SharedKernel" Version="2.1.0" />
+    <PackageVersion Include="Ardalis.SharedKernel" Version="1.6.0" />
```

Comment:
I didn't have this version, so used the latest NuGet version. I added the missing types with no implementation.

Reply:
Yeah I only have that 2.1 version locally where I was just messing around with different approaches to the EntityBase class.